src/jwe.c

```diff
@@ -3332,6 +3332,24 @@ int r_jwe_encrypt_payload(jwe_t * jwe) {
   return ret;
 }
 
+static int gnutls_is_block_cipher(gnutls_cipher_algorithm_t alg)
```

Comment:
I would rename this function `_r_gnutls_is_block_cipher` to be more consistent with the other static function names

Reply:
done